### DISCUSSION TITLE
Help on Deploying to Netlify

### DISCUSSION THREAD

**Discussion by xditya** [fastapi/fastapi#3064] [Questions] [answered]
Can someone please guide me on deploying a simple FastAPI app to netlify! Would be grateful. :)

**Comment by hayruni** [comment]
My

**Comment by hayruni** [comment]
- [ ]

**Comment by Kludex** [comment]
What do you want to know?

**Reply by xditya** [reply]
Just some basic steps to get fastapi running on netlify.

**Reply by Kludex** [reply]
If you can't find any resource about FastAPI on Netlify, any Python project will do. Try to research for "Flask deploy Netlify". Then you can adapt.

My answer here is just for you to not wait long for people, as it has already been 6 days.

**Reply by xditya** [reply]
I've done quite a few research and ended up in nothing 
Thanks for the reply tho, I'll find out someway to sort my problem out.

**Comment by kbakk** [comment, marked as answer]
Netlify doesn't appear to support Python: 

> Currently, you can deploy serverless functions built with JavaScript and Go. 

https://docs.netlify.com/functions/configure-and-deploy/#languages-and-language-settings

**Reply by xditya** [reply]
👍 thanks

**Reply by kbakk** [reply]
@xditya From the documentation, Netlify appears to just be a wrapper around AWS Lambda, and FastAPI does work on AWS Lambda – see for example [this guide](https://iwpnd.pw/articles/2020-01/deploy-fastapi-to-aws-lambda)

**Reply by xditya** [reply]
> @xditya From the documentation, Netlify appears to just be a wrapper around AWS Lambda, and FastAPI does work on AWS Lambda – see for example [this guide](https://iwpnd.pw/articles/2020-01/deploy-fastapi-to-aws-lambda)

Oh, thanks :)
I'll see to it. 👍🏻
